tedana/workflows/parser_utils.py

```diff
@@ -4,6 +4,26 @@
 import os.path as op
 import logging
 
+import argparse
+
+
+def string_or_float(string):
```

Comment:
This is really nitpicky but I feel like you're not checking "string or float," you're checking "string or fraction." For example, 1.1 would throw an error here, even though 1.1 is a completely valid float. So out of specificity I think we should consider renaming this (I'm not sure that "fraction" is the best descriptor either).

Reply:
Since it's not a generic function I think naming it `string_or_float` was a mistake. I was originally trying to see if I could feed in the valid options from the parser rather than this function, but since that's not the case and the valid options are hardcoded, what about `check_tedpca_value`?

Reply:
That sounds much better to me, yes!